Test error:

```
12:28:24 [ERROR] tensor_add_update_on_not_fully_specified_cell_throws(com.yahoo.document.json.JsonReaderTestCase)  Time elapsed: 0.005 s  <<< FAILURE!
12:28:24 org.junit.ComparisonFailure: expected:<...label for dimension [y] for tensor(x{},y{})> but was:<...label for dimension ['y'] for tensor(x{},y{})>
12:28:24 	at org.junit.Assert.assertEquals(Assert.java:117)
12:28:24 	at org.junit.Assert.assertEquals(Assert.java:146)
12:28:24 	at com.yahoo.document.json.JsonReaderTestCase.illegalTensorAddUpdate(JsonReaderTestCase.java:1946)
12:28:24 	at com.yahoo.document.json.JsonReaderTestCase.tensor_add_update_on_not_fully_specified_cell_throws(JsonReaderTestCase.java:1760)
```

--- a/document/src/test/java/com/yahoo/document/json/JsonReaderTestCase.java
+++ b/document/src/test/java/com/yahoo/document/json/JsonReaderTestCase.java
@@ -1757,7 +1757,7 @@ public class JsonReaderTestCase {
 
     @Test
     public void tensor_add_update_on_not_fully_specified_cell_throws() {
-        illegalTensorAddUpdate("Error in 'sparse_tensor': Missing a label for dimension y for tensor(x{},y{})",
+        illegalTensorAddUpdate("Error in 'sparse_tensor': Missing a label for dimension 'y' for tensor(x{},y{})",
                                "sparse_tensor",
                                "{",
                                "  'cells': [",

